tests/console/system_prepare.pm

```diff
@@ -87,7 +88,10 @@ sub run {
             diag "SUSEConnect --status-text locked: $out";
         }
         diag "SUSEConnect --status-text: $out";
-        assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'" unless get_var('MEDIA_UPGRADE');
+        if (!get_var('MEDIA_UPGRADE')) {
+            assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'";
+            services::registered_addons::full_registered_check;
```

Comment:
I don't think that we have clash of names, so it should work if we just use `full_registered_check`. Otherwise we can skip explicit import as we call it by full name.

Reply:
I used 'full_registered_check' before, it cannot find this file. So I change to use full path

Reply:
I've checked now, and it's actually because it's not exported in the `lib/services/registered_addons.pm`. As you import that method, I would propose to export it properly in `lib/services/registered_addons.pm`

Reply:
Hi Rodion, I merge it seems early, do you think we need revert it and add this modification?